config/config.go

```diff
@@ -28,6 +28,7 @@ var (
 		TLSMinVersStr string   `yaml:"tls-min-vers"`
 		URLPrefix     string   `yaml:"url-prefix"`
 		Referrers     []string `yaml:"referrers"`
+		AccessKey     string   `yaml:"accessKey"`
```

Comment:
`yaml:"accessKey"` -> `yaml:"access-key"`
This is also required to match the documentation in "README.md".